src/main/java/com/alkemy/ong/data/entity/CategoryEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.alkemy.ong.data.entity;
```

Comment:
falta el script en schema.sql de la tabla.
CREATE TABLE IF NOT EXISTS table name